webui-src/app/people/people_ownids.js

```diff
@@ -1,213 +1,379 @@
-      m('i.fas.fa-user-plus'),
-      m('h3', 'Create new Identity'),
-      m('hr'),
-      m('input[type=text][placeholder=Name]', {
+    view: (v) => [
+      m("i.fas.fa-user-plus"),
+      m("h3", "Create new Identity"),
+      m("hr"),
+      m("input[type=text][placeholder=Name]", {
         value: name,
-        oninput: e => (name = e.target.value),
+        oninput: (e) => (name = e.target.value),
       }),
-      'Type:',
       m(
-        'select',
+        "div",
         {
-          value: pseudonimous,
-          oninput: e => (type = e.target.value),
+          style: "display:inline; margin-left:5px;",
         },
         [
-          m('option[value=false]', 'Linked to your Profile'),
-          m('option[value=true]', 'Pseudonymous'),
-        ],
+          "Type:",
+          m(
+            "select",
+            {
+              value: pseudonimous,
+              style: "border:1px solid black",
+              oninput: (e) => {
+                if (e.target.value == "false") pseudonimous = false;
+                else pseudonimous = true;
+              },
+            },
+            [
+              m("option[value=false]", "Linked to your Profile"),
+              m("option[value=true]", "Pseudonymous"),
+            ]
+          ),
+        ]
       ),
-      m('br'),
-      m('input[type=password][placeholder=Retroshare Passphrase]', {
+      m("br"),
+      m("input[type=password][placeholder=Retroshare Passphrase]", {
         value: pgpPassword,
-        oninput: e => (pgpPassword = e.target.value),
+        oninput: (e) => (pgpPassword = e.target.value),
       }),
       m(
-        'p',
-        'You can have one or more identities. ' +
-          'They are used when you chat in lobbies, ' +
-          'forums and channel comments. ' +
-          'They act as the destination for distant chat and ' +
-          'the Retroshare distant mail system.',
+        "p",
+        "You can have one or more identities. " +
+          "They are used when you chat in lobbies, " +
+          "forums and channel comments. " +
+          "They act as the destination for distant chat and " +
+          "the Retroshare distant mail system."
       ),
       m(
-        'button',
+        "button",
         {
-          onclick: () =>
-            rs.rsJsonApiRequest(
-              '/rsIdentity/createIdentity',
-              {
-                name,
-                pseudonimous,
-                pgpPassword,
-              },
-              data => {
-                const message = data.retval
-                  ? 'Successfully created identity.'
-                  : 'An error occured while creating identity.';
-                widget.popupMessage([
-                  m('h3', 'Create new Identity'),
-                  m('hr'),
-                  message,
-                ]);
-                m.redraw();
-              },
-            ),
+          onclick: () => {
+            if (pseudonimous == false) {
```

Comment:
Use strict equals.

Reply:
@rottencandy, Pseudonymous is a boolean data type. It also works without strict equals. But I will change it.

Reply:
If it's already boolean, you don't have to use equals at all. Can simply do
```js
if (!pseudonimous) {
```

Reply:
Thanks @rottencandy , i will change it